apps/web-portal/lib/use-cases/auth/AuthLogin.mjs

```diff
@@ -2,12 +2,12 @@ import {UseCase} from '@zorko-io/util-use-case'
 import {apiToken} from '../utils'
 
 export class AuthLogin extends UseCase {
-  // eslint-disable-next-line no-unused-vars
-  async run(params) {
-    if (params.token) {
-      apiToken.verify(params.token)
-    }
+  static rules = {
```

Comment:
Could you double check LIVR format for declaration of both (email/password) or (token) payload , I see the code duplication between both UseCases, and only rules are different

Reply:
I also think about such LIVR rules, but unfortunately, there are no embedded rules. We can create our own but not sure about it.

Reply:
What about this one https://livr-spec.org/validation-rules/metarules.html#variable-object ?

Reply:
This rule is not working, at least I could not found

Reply:
Could you add TODO for finding a way to define such rule with LIVR ?

Reply:
add 'or' rule